features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/Environments.jsx

```diff
@@ -57,188 +272,1561 @@ const useStyles = makeStyles((theme) => ({
  */
 export default function Environments() {
```

Comment:
We should change the name here to reflect the page name I:e `Deployments`


and also split the component into multiple sub components

![image](https://user-images.githubusercontent.com/3313885/105878321-eca21100-6026-11eb-9e29-54a52ea9658e.png)


like, below

Reply:
![image](https://user-images.githubusercontent.com/3313885/105878999-aef1b800-6027-11eb-9aa5-5cb3d6f4b809.png)